lib/Listener/UserDeletedListener.php

```diff
@@ -52,13 +52,18 @@ public function handle(Event $event): void {
 		$user = $event->getUser();
 
 		$rooms = $this->manager->getRoomsForUser($user->getUID());
-
 		foreach ($rooms as $room) {
 			if ($this->participantService->getNumberOfUsers($room) === 1) {
 				$room->deleteRoom();
 			} else {
 				$this->participantService->removeUser($room, $user, Room::PARTICIPANT_REMOVED);
 			}
 		}
+
+		$leftRooms = $this->manager->getLeftOneToOneRoomsForUser($user->getUID());
+		foreach ($leftRooms as $room) {
+			$room->setType(Room::GROUP_CALL, true);
+			$room->setName($user->getDisplayName(), '');
```

Comment:
add a comment to explain what this implies: that the room won't be found any more by name when trying to recreate the same 1-1 conversation

Reply:
Added